proto/authority/policies.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package zetachain.zetacore.authority;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/zeta-chain/zetacore/x/authority/types";
+
+// PolicyType defines the type of policy
+enum PolicyType {
+  option (gogoproto.goproto_enum_stringer) = true;
+  groupEmergency = 0;
+  groupAdmin = 1;
+}
+
+message PolicyAddress {
+  PolicyType policy_type = 1;
```

Comment:
Any reason you want to keep the policy type , inside the policy address? imho , it seems easier to understand if a policy has only 1 type